conduct/conduct_reporting_online.md

```diff
@@ -0,0 +1,62 @@
+# Project Jupyter Code of Conduct - Reporting Guide - Online Community
+
+If you believe someone is violating the code of conduct we ask that you report
+it to Project Jupyter by emailing
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org) or by completing the
+following form [this form](https://goo.gl/forms/sJzOIie3zde9M71T2). All reports
+will be kept confidential. In some cases we may determine that a public
+statement will need to be made. If that's the case, the identities of all
+involved will remain confidential unless those individuals instruct us
+otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate law
+enforcement first. If you are unsure what law enforcement agency is appropriate,
+please include this in your report and we will attempt to notify them.
+
+If you choose to email instead of using the form, please include the following
+information:
+
+* Your contact info (not required, if you would like to remain anonymous please
+  use the form)
+* Names (real, nicknames, or pseudonyms) of any individuals involved. If there
+  were other witnesses besides you, try to include them as well
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. If there is a publicly available record (e.g. a
+  mailing list archive or GitHub/gitter.im) please include a link
+* Any extra context you believe exists for the incident
+* If you believe this incident is ongoing
+* Any other information you believe we should have
+
+
+## What happens after you file a report?
+
+You will receive an email from the Project Jupyter Code of Conduct Committee
+acknowledging receipt immediately. We promise to acknowledge receipt within 24
+hours (and will aim for much quicker than that).
+
+The working group will immediately meet to review the incident and determine:
+* What happened
+* Whether this event constitutes a Code of Conduct violation
+* Who the individual(s) involved were
+* Whether this is an ongoing situation, or if there is a threat to anyone's
+  physical safety
+
+If this is determined to be an ongoing incident or a threat to physical safety,
+the committee's immediate priority will be to protect everyone involved. This
+means we may delay an "official" response until we believe that the situation
+has ended and that everyone is physically safe.
+
+The CoC committee will then follow the
+[standard procedure][conduct_enforcement.md#Resolutions] to arrive at and
+communicate a resolution.
+
+## Appealing the Code of Conduct Committee’s Response
+
+Only permanent resolutions (such as bans) may be appealed. To appeal a decision
+of the Code of Conduct Committee, contact the Jupyter Steering Council at
+[*steeringcouncil@jupyter.org*](mailto:steeringcouncil@jupyter.org) with your
+appeal and they will review the case.
+
+The Committee should discuss the appeal considering any additional information
```

Comment:
This appears to suggest that the CoC "committee" will review the appeal. But in the previous sentence, we say that the SC will handle the appeal. My own preference is for the SC to make any decisions about the appeal, but by first gathering any/all information from the CoC committee.